portfolio/src/main/webapp/script.js

```diff
@@ -13,19 +13,30 @@
 // limitations under the License.
 
 async function getPhrase(){
-    let response = await fetch('/load-data');
-    let list = await response.json();
-    if(list.length != 0){
-        for(i = 0; i < list.length; i++){
-            let commentSection = document.createElement("DIV");
-            commentSection.setAttribute("id", "dynamic-history");
-            let addName = document.createTextNode(list[i].name + ": ");
-            let addMessage = document.createTextNode(list[i].message);
-            commentSection.appendChild(addName);  
-            commentSection.appendChild(addMessage);  
-            document.getElementById("section").appendChild(commentSection);
-        }
+    let max = document.getElementById("max-comments").value;
+    let response = await fetch('/load-data?numCommentsDisplay=' + max);
+    let list = await response.json();//list of entities from datastore
+    let commentHistory = document.getElementById("section");//UI for displaying comments
```

Comment:
Put a space between the code and the comments

Reply:
I'll add spaces